GitHub actions failing

All five of the last PRs that run checks (four of which are mine) fail on the MacOS side:

* https://github.com/MonoGame/MonoGame/pull/8015
* https://github.com/MonoGame/MonoGame/pull/8018
* https://github.com/MonoGame/MonoGame/pull/8019
* https://github.com/MonoGame/MonoGame/pull/8007
* https://github.com/MonoGame/MonoGame/pull/8014

There are a couple of other PRs that don't have any actions to run that didn't have the issue.

In all of these cases, the linux side succeeded, the mac side errored out, and the windows side was cancelled after the mac side failed, since it was still running.

All of them show the same stuff in the output. Here's an example: https://github.com/MonoGame/MonoGame/actions/runs/4706087874/jobs/8347094062?pr=8015

As far as I can tell, this looks like it might be the first "real" issue:

```
========================================
PackVSMacTemplates
========================================
  Determining projects to restore...
  Restored /Users/runner/work/MonoGame/MonoGame/Templates/MonoGame.Templates.VSMacExtension/MonoGame.Templates.VSMacExtension.csproj (in 3.27 sec).
MSBuild version 17.5.0+6f08c67f3 for .NET
  Determining projects to restore...
  Restored /Users/runner/work/MonoGame/MonoGame/Templates/MonoGame.Templates.VSMacExtension/MonoGame.Templates.VSMacExtension.csproj (in 402 ms).
/Users/runner/.nuget/packages/microsoft.visualstudiomac.sdk/17.0.0/build/Microsoft.VisualStudioMac.Sdk.targets(55,5): warning : The TargetFrameworkVersion 'v6.0' does not the target instance. Overriding with 'v7.0'. [/Users/runner/work/MonoGame/MonoGame/Templates/MonoGame.Templates.VSMacExtension/MonoGame.Templates.VSMacExtension.csproj]
```
This part seems problematic:
> TargetFrameworkVersion 'v6.0' does not the target instance. Overriding with 'v7.0'

After that, there's a long collection of things like this:

> warning MSB3243: No way to resolve conflict between "Microsoft.Win32.SystemEvents, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" and "Microsoft.Win32.SystemEvents, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51". Choosing "Microsoft.Win32.SystemEvents, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" arbitrarily.

After a giant pile of those, we start to get errors like this:

> error CS0518: Predefined type 'System.Object' is not defined or imported

#### What version of MonoGame does the bug occur on:
- MonoGame 3.8.1, but this doesn't seem like an MG source code issue, but some sort of build action configuration issue.

#### What operating system are you using:
- MacOS? I used Windows during development, but the builds are dying on the macos side.

#### What MonoGame platform are you using:
<!-- e.g. DesktopGL, WindowsDX, WindowsUWP, Android -->
- Not relevant.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -18,6 +18,10 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: '6.0.x'
+      - name: Setup .NET Core SDK ${{ matrix.dotnet-version }}
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '7.0.x'
       - name: Setup Java
         uses: actions/setup-java@v2
         with:
@@ -62,7 +66,7 @@ jobs:
         with:
           name: Visual Studio for Mac Extension
           path: |
-            Artifacts/MonoGame.Templates.VSMacExtension/Release/net6.0/*.mpack
+            Artifacts/MonoGame.Templates.VSMacExtension/Release/**/*.mpack
           if-no-files-found: error
   deploy:
     name: deploy

--- a/Templates/MonoGame.Templates.VSMacExtension/MonoGame.Templates.VSMacExtension.csproj
+++ b/Templates/MonoGame.Templates.VSMacExtension/MonoGame.Templates.VSMacExtension.csproj
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <BaseOutputPath>..\..\Artifacts\MonoGame.Templates.VSMacExtension</BaseOutputPath>
     <CreatePackage>true</CreatePackage>
     <ImplicitUsings>true</ImplicitUsings>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudioMac.Sdk" Version="17.0.0" />
+    <PackageReference Include="Microsoft.VisualStudioMac.Sdk" Version="17.5.0" />
   </ItemGroup>
 
   <ItemGroup>

